zip-0227.rst

```diff
@@ -1,8 +1,359 @@
 ::
 
-  ZIP: 226
-  Title: Zcash Shielded Assets - Issuance
-  Owners: Daniel Bennaroch
-  Status: Reserved
+  ZIP: 227
+  Title: Issuance of Zcash Shielded Assets
+  Owners: Daniel Benarroch <daniel@qed-it.com>
+          Pablo Kogan <pablo@qed-it.com>
+          Aurelien Nicolas <aurel@qed-it.com>
+  Credits: Daira Hopwood
+           Jack Grigg
+           Vivek Arte
+  Status: Draft
   Category: Consensus
+  Created: 2022-05-01
+  License: MIT
   Discussions-To: <https://github.com/zcash/zips/issues/618>
+ 
+Terminology
+===========
+
+The key word "MUST" in this document is to be interpreted as described in RFC 2119 [#RFC2119]_.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_.
+
+The term "Orchard" in this document is to be interpreted as described in ZIP 224 [#zip-0224]_.
+
+We define the following additional terms:
+
+- Custom Asset: A type of note that can be transferred on the Zcash block chain. ZEC is the default (and currently the only defined) asset for the Zcash mainnet. Different custom assets are defined and differentiated by their identifier (`AssetId`).
+- Zcash Shielded Asset: an asset with issuance defined on the Zcash block chain, and, with this ZIP, belonging to the Orchard anonymity pool.
+- Wrapped Asset: a ZSA asset with native issuance defined outside the Zcash block chain.
+- Issuance Action: an instance of a single issuance of a Zcash Shielded Asset. It defines the issuance of a single asset identifier.
+- Issuance Bundle: the bundle in the transaction that contains all the issuance actions of that transaction.
+
+Abstract
+========
+
+ZIP 226 [#zip-0226]_ and ZIP 227 [#zip-0227]_ propose in conjunction the Zcash Shielded Assets (ZSA) protocol, which is an extension of the Orchard protocol that enables the creation, transfer and burn of custom assets on the Zcash chain. The creation of such assets is defined in ZIP 227 [#zip-0227]_. The transfer and burn of such assets is defined in ZIP 226 [#zip-0226]_. This ZIP must only be implemented in conjuction with ZIP 226 [#zip-0226]_, as the issuance mechanism is only valid for the ZSA transfer protocol, as it produces notes that can only be transferred under ZSA.
+
+Motivation
+==========
+
+The current Orchard protocol does not support custom assets. Enabling multi-asset support on the Zcash chain will open the door for a host of applications and enhance the ecosystem with application developers and asset custody institutions for issuance and
+bridging purposes.
+
+Use Cases
+=========
+
+The design presented in this ZIP enables issuance of shielded assets in various modes:
+
+- The issuer knows in advance the receivers of the issued asset.
```

Comment:
```suggestion
- The issuer may or may not know the receivers of the issued asset in advance.
```

Reply:
We suggest that every asset is always issued into a specific address. If you issue to yourself, we still call this "receiver is known". Did you have another use in mind?

Reply:
So what you're saying is "the address of the receiver is always known in advance".

Does that statement belong in the requirements or consequences of the design, not in a section that specifically provides alternative modes?
(If there is no alternative, then "issuance of shielded assets in various modes" doesn't really apply.)

However, if you meant "the identity of the receiver is always known in advance", then here are some example use cases: - an address that is cryptographically impossible to spend from,
- an address that depends on solving a specific calculation, or
- a multi-signature address where most of the parties aren't known to the issuer.

Reply:
That makes sense, yes, thanks! Adding in the suggestion.